jupyterhub_singleuser_profiles/profiles.py

```diff
@@ -75,7 +76,7 @@ def get_config_maps_matching_label(self, target_label='jupyterhub=singleuser-pro
     _LOGGER.info("Found these additional Config Maps: %s" % config_maps_list)
     return config_maps_list
 
-  def read_config_map(self, config_map_name, key_name="profiles"):
+  def read_config_map(self, config_map_name, key_name="profile"):
```

Comment:
Will this break loading of profiles? Maybe we should remove the default and expect every function to provide the `key_name` when calling this method

Reply:
All original calls of this function seem to use set key_name to "profile", so those should not break, and mine do not set it, and use the modified default. I feel like keeping the default value there is safer.